spec/models/authenticator/ldap_spec.rb

```diff
@@ -94,10 +94,14 @@ def normalize(dn)
   end
 
   before(:each) do
-    allow(MiqLdap).to receive(:new).and_return { FakeLdap.new(user_data) }
```

Comment:
@jrafanie HAH. Figured this out now. `.and_return {}` is deprecated, `.and_return()` is not. `.and_return{ }` => just `{ }`, `and_return()` => (keep)

:unamused: :sparkles: